### PR TITLE
Clean up comments

### PR DESCRIPTION
**Pull request by chrisberthe**
Updates the SCSS comments to follow these conventions:

1. `// whatever`: simple one line comment
2. `/** ... */`: larger comment that requires multiple lines

@t-kelly @tauthomas01

### CODE REVIEW

File: src/assets/styles/tools/mixins.scss [hunk 1]
```diff
@@ -1,34 +1,16 @@
-/* stylelint-disable declaration-no-important */
-
-/* ================ Mixins ================ */
+// stylelint-disable declaration-no-important
```
Comment (tauthomas01) [inline src/assets/styles/tools/mixins.scss]: I'm curious. Can you make a test and see if it works with those comment style? I'm checking stylelint documentation and it looks like they use `/* */` one : https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md

File: src/assets/styles/tools/mixins.scss [hunk 2]
```diff
@@ -1,34 +1,16 @@
...
 @mixin clearfix() {
   &::after {
     content: '';
     display: table;
     clear: both;
   }
 
-  // sass-lint:disable;
+  // stylelint-disable-next-line
```
Comment (tauthomas01) [inline src/assets/styles/tools/mixins.scss]: Same comment as above

File: src/assets/styles/tools/mixins.scss (hunk 1 above)
Comment (chrisberthe) [reply src/assets/styles/tools/mixins.scss]: I had checked the documentation too and it works okay :') can you try on your side just to make sure?

Comment (tauthomas01) [reply src/assets/styles/tools/mixins.scss]: It works fine on my end as well, thanks! 👍

**Pull request merged**